.github/workflows/lint.yml

```diff
@@ -15,3 +15,15 @@ jobs:
       with:
         files: .
         config_file: ".markdownlint.yaml"
+  docker:
+    runs-on: ubuntu-latest
+    steps: 
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: dockerlint
+      uses: brpaz/hadolint-action@v1.1.0
+      with:
+        dockerfile: docker/Dockerfile
+      env:
```

Comment:
Are these environment variables required? The target is already specified in the `with` clause and the secret token shouldn't be needed for a public repository.

Reply:
You're right. Removed :-)